perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -5,7 +5,22 @@
 import java.util.HashSet;
```

Comment:
I just noticed that this class has its interface class empty. Should I create it or just write it down for later fix? Or maybe it is intensionaly...